test/helpers.lua

```diff
@@ -297,12 +297,13 @@ local function dedent(str)
     -- no minimum common indent
     return str
   end
+  local left_indent = (' '):rep(leave_indent or 0)
```

Comment:
With `gsub(..., 0)` nothing happens. What is the purpose of calling dedent() if the indent is not un-indented?